src/lib/browse/LayerControls.svelte

```diff
@@ -85,14 +86,15 @@
     const previousSessionQueryString =
       window.localStorage.getItem(localStorageQuerystringKey) || "";
     const newQueryParams = new URLSearchParams(previousSessionQueryString);
-    // @ts-ignore typescript is convinced foreach doesn't exist but it's working?
-    newQueryParams.entries().forEach(([key, value]) => {
+    for (const entry in newQueryParams.entries) {
```

Comment:
Destructuring works too, `for let ([key, value] of newQueryParams.entries())`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

And I don't think you want [for in](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in)

And [entries](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/entries) is a method, does `in x.entries` work? (and TS not complain?!)